vulnerabilities/importers/mozilla.py

```diff
@@ -0,0 +1,208 @@
+from typing import Set, List, Generator
+
+import re
+import asyncio
+from bs4 import BeautifulSoup
```

Comment:
Group your imports into 

1. Stdlib imports
2. 3rd party imports
3. project imports

Separate them by one line and sort them.

Reply:
On it